ngx_rtmp_live_module.c

```diff
@@ -228,16 +228,37 @@ ngx_rtmp_live_free_message(ngx_rtmp_session_t *s, ngx_chain_t *in)
 }
 
 
+static void
+ngx_rtmp_live_free_pool_cleanup(void *data)
+{
+    ngx_rtmp_live_app_conf_t      *lacf = data;
+
+    if (lacf->pool != NULL) {
+        ngx_destroy_pool(lacf->pool);
+        lacf->pool = NULL;
+    }
+}
+
+
 static void *
 ngx_rtmp_live_create_app_conf(ngx_conf_t *cf)
 {
+    ngx_pool_cleanup_t            *cln;
     ngx_rtmp_live_app_conf_t      *lacf;
 
     lacf = ngx_pcalloc(cf->pool, sizeof(ngx_rtmp_live_app_conf_t));
     if (lacf == NULL) {
         return NULL;
     }
 
+    cln = ngx_pool_cleanup_add(cf->pool, 0);
```

Comment:
Placing these code snippets in `ngx_rtmp_merge_app_conf` will fix the memory leak?
Doing so keeps style consistent to the above one.